src/style.css

```diff
@@ -1,3 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+background-color: gainsboro;
+}
+
+body, .header_nav a {
+  font-family: 'Roboto', sans-serif;
+}
+
+.header_top {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 20px;
+}
+
+.header_section {
+  justify-content: flex-end;
+  margin: 0 50px;
+}
+
+.header_logo img{
+  margin-left: 50px;
+  align-items: center;
+}
+
+.header_nav {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
   margin: 0;
 }
+
+.header_nav a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav_item {
+  margin-right: 20px;
+  text-transform: uppercase;
+  font-size: 12px
```

Comment:
The `font-size` property is missing a semicolon at the end. It's important to end each declaration with a semicolon for proper CSS syntax.